pkg/capabilities/triggers/mercury_trigger_test.go

```diff
@@ -13,43 +16,56 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
-var (
-	feedOne   = "0x1111111111111111111100000000000000000000000000000000000000000000"
-	feedTwo   = "0x2222222222222222222200000000000000000000000000000000000000000000"
-	feedThree = "0x3333333333333333333300000000000000000000000000000000000000000000"
-	feedFour  = "0x4444444444444444444400000000000000000000000000000000000000000000"
-	feedFive  = "0x5555555555555555555500000000000000000000000000000000000000000000"
-)
-
-func TestMercuryTrigger(t *testing.T) {
-	ts := NewMercuryTriggerService(100, logger.Nop())
-	ctx := tests.Context(t)
+func registeredTriggerHelper(
```

Comment:
@HenryNguyen5 Can we call this something more descriptive that says what it does, eg. registerTrigger?

Reply:
Yeah, this deserves some comments